website/src/docs/vue.md

```diff
@@ -0,0 +1,151 @@
+---
+title: "Vue Integration"
+type: docs
+module: "@uppy/vue"
+permalink: docs/vue/
+order: 0
+category: "Vue"
+---
+
+Uppy provides [Vue][] components for the included UI plugins.
+
+## Installation
+
+All Vue components are provided through the `@uppy/vue` package
+
+Install from NPM:
+
+```sh
+npm install @uppy/vue
+```
+## CSS
+
+Make sure to also include the necessary CSS files for each Uppy Vue component you are using.
+
+## Usage
+
+The components can be used with [Vue][] and frameworks that use it, like [Nuxt][].
+
+Instead of adding a UI plugin to an Uppy instance with `.use()`, the Uppy instance can be passed into components as an `uppy` prop. 
+
+Note: At the moment, this example is not functional due to an unresolved issue with using other plugins with the Dashboard
```

Comment:
Still the case? 🤔

Reply:
Oh no I need to remove that. Plugins work fine now, just an artifact I forgot to get rid of